static/app/components/tabs/tab.tsx

```diff
@@ -47,7 +48,13 @@ function BaseTab(
       <HoverLayer orientation={orientation} />
       <FocusLayer orientation={orientation} />
       {rendered}
-      <TabSelectionIndicator orientation={orientation} selected={isSelected} />
+      {isSelected && (
+        <TabSelectionIndicator
+          layoutId="indicator"
+          transition={{type: 'spring', stiffness: 250, damping: 25}}
```

Comment:
I like how this makes the animation feel more speedy, but as you said the spring may affect the performance. Could we remove the spring effect (where the indicator overshoots and then bounces back) but still make it feel as speedy?

Reply:
yeah let me play with it a bit more later